examples/Protobuf/cmake/Hunter/config.cmake

```diff
@@ -0,0 +1,14 @@
+if (ANDROID OR IOS)
+    hunter_config(Protobuf
+            VERSION ${HUNTER_Protobuf_VERSION}
+            CMAKE_ARGS
+                protobuf_BUILD_PROTOC=OFF
+                protobuf_BUILD_LIBPROTOBUF_LITE=OFF
+    )
+else()
+    hunter_config(Protobuf
+            VERSION ${HUNTER_Protobuf_VERSION}
+            CMAKE_ARGS
+                protobuf_BUILD_LIBPROTOBUF_LITE=OFF
+    )
+endif()
```

Comment:
http://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file